src/components/Footer.js

```diff
@@ -0,0 +1,12 @@
+const Footer = () => {
```

Comment:
good use of making footer as component instead of coding it on html. Seeing this just shows that you're very confident using react! I'll take note of this for my project!!